docs/src/pages/docs/functions/logic-functions.md

```diff
@@ -308,6 +442,56 @@ function. Either the original or modified value will be returned depending on
 the result of the predicate. Check out [`unless`](#unless) for a negated version
 of this function.
 
+```javascript
+import when from 'crocks/logic/when'
+
+import constant from 'crocks/combinator/constant'
+import flip from 'crocks/combinator/flip'
+import composeB from 'crocks/combinator/composeB'
+import compose from 'crocks/core/compose'
+
+// prod :: Number -> Number -> Number
+const prod = a => b => b * a
```

Comment:
This is only used in ALL of these examples to double. Thinking is may be easier to follow if this was just a double function for all of these eamples that use `prod`. sometghing like:
```js
// double :: Number -> Number
const double =
  x => x * 2
```
thoughts?

Reply:
yeah makes sense. I mostly do it out of habit, but i prefer the simplicity